components/docs-chef-io/content/automate/major_upgrade.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Upgrade Chef Automate to 3.x.x"
+date = 2022-03-03T12:02:46-08:00
+draft = false
+
+[menu]
+  [menu.automate]
+    title = "Upgrade to 3.x.x"
+    identifier = "automate/major_upgrade.md Major Upgrade"
+    parent = "automate"
++++
+
+[\[edit on GitHub\]](https://github.com/chef/automate/blob/main/components/docs-chef-io/content/automate/major_upgrade.md)
+
+Chef Automate is a platform that provides an entire suite of enterprise capabilities for node visibility and compliance. Chef Automate upgrades from one minor version to another automatically, but you cannot directly upgrade to any major version of Chef Automate. Use the `--major` flag to upgrade from the latest to a major version. This section will talk about the major version upgrade of Chef Automate with its possible scenarios.
+
+## Upgrade Journey
+
+Please choose following upgrade journey based on your current version of chef automate.All these upgrades are mannual upgrades.
+
+| Your Current Version | Upgrade To |
+| -------------------- | ---------- |
+| Any version before 2022012400000| 2022012400000|
+| on 2022012400000| 3.0.1|
+| 3.0.1| 4.0.1|
```

Comment:
```suggestion
```
Not required as this upgrade talks about upgrade till version 3 and not beyond that